<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Caisse</class>
 <widget class="QDialog" name="Caisse">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="status">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#status { border-color:#cccccc; border-style:solid; font-size:14px; padding:4px; border-radius:10px; border-width:1px; box-shadow: 0px 0px 5px 0px rgba(42,42,42,.75);  } 
		 .css-input:focus { outline:none; } </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="round">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>dot</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>460</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">width: 385px;
height: 45px;
background-color: #efeef8;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
width: 60px;
height: 60px;

background-color: rgba(255, 255, 255, 0.9);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>347</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
width: 345px;
height: 60px;
border-radius: 6px;
border: 2px solid #cadee7;
background-color: rgba(255, 255, 255, 0.9);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>347</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
width: 345px;
height: 60px;
border-radius: 6px;
border: 2px solid #cadee7;
background-color: rgba(255, 255, 255, 0.9);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>290</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
width: 345px;
height: 60px;
border-radius: 6px;
border: 0px solid #cadee7;

background-color: rgba(255, 255, 255, 0.9);</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
width: 60px;
height: 60px;

background-color: rgba(255, 255, 255, 0.9);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">width: 345px;
height: 19px;
color: #335b68;
font-family: &quot;SF UI Display&quot;;
font-size: 16px;
font-weight: 500;
/* Text style for &quot;User nam&quot; */
letter-spacing: 0.8px;</string>
   </property>
   <property name="text">
    <string>EMAIL</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">width: 345px;
height: 19px;
color: #335b68;
font-family: &quot;SF UI Display&quot;;
font-size: 16px;
font-weight: 500;
/* Text style for &quot;Passwor&quot; */
letter-spacing: 0.8px;</string>
   </property>
   <property name="text">
    <string>PASSWORD</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>321</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="styleSheet">
       <string notr="true">#pushButton_3 {
	background-image:/img/Rectangle_4.png;
width: 101px;
height: 48px;
border-radius: 4px;
background-color: #5e77ff;


color: #ffffff;
font-family: &quot;SF UI Display&quot;;
font-size: 16px;
font-weight: 600;
line-height: 20px;
/* Text style for &quot;Login&quot; */
letter-spacing: 0.5px;
}

#pushButton_3:hover {
	background:-webkit-gradient(linear, left top, left bottom, color-stop(0.05, #019ad2), color-stop(1, #33bdef));
	background:-moz-linear-gradient(top, #019ad2 5%, #33bdef 100%);
	background:-webkit-linear-gradient(top, #019ad2 5%, #33bdef 100%);
	background:-o-linear-gradient(top, #019ad2 5%, #33bdef 100%);
	background:-ms-linear-gradient(top, #019ad2 5%, #33bdef 100%);
	background:linear-gradient(to bottom, #019ad2 5%, #33bdef 100%);

	background-color:#019ad2;
}
#pushButton_3:active {
	position:relative;
	top:1px;
}</string>
      </property>
      <property name="text">
       <string>Forgot Password?</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="styleSheet">
       <string notr="true">#pushButton_2 {
	background-image:/img/Rectangle_4.png;
width: 101px;
height: 48px;
border-radius: 4px;
background-color: #5e77ff;


color: #ffffff;

font-family: &quot;SF UI Display&quot;;
font-size: 18px;
font-weight: 600;
line-height: 20px;
/* Text style for &quot;Login&quot; */
letter-spacing: 0.5px;}

#pushButton_2:hover {
	background:-webkit-gradient(linear, left top, left bottom, color-stop(0.05, #019ad2), color-stop(1, #33bdef));
	background:-moz-linear-gradient(top, #019ad2 5%, #33bdef 100%);
	background:-webkit-linear-gradient(top, #019ad2 5%, #33bdef 100%);
	background:-o-linear-gradient(top, #019ad2 5%, #33bdef 100%);
	background:-ms-linear-gradient(top, #019ad2 5%, #33bdef 100%);
	background:linear-gradient(to bottom, #019ad2 5%, #33bdef 100%);

	background-color:#019ad2;
}
#pushButton_2:active {
	position:relative;
	top:1px;
}</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>290</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
width: 345px;
height: 60px;
border-radius: 6px;
border: 0px solid #cadee7;

background-color: rgba(255, 255, 255, 0.9);</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>20</y>
     <width>161</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>logo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_12">
      <property name="styleSheet">
       <string notr="true">width: 300px;
height: 96px;
color: #85b9ca;
font-family: &quot;SFUI Display - Heavy&quot;;
font-size: 20px;
font-weight: 800;
/* Text style for &quot;Welcome  B&quot; */
letter-spacing: 0.6px;</string>
      </property>
      <property name="text">
       <string>Welcome Back </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="device_id">
      <property name="styleSheet">
       <string notr="true">width: 300px;
height: 96px;
color: #85b9ca;
font-family: &quot;SFUI Display - Heavy&quot;;
font-size: 20px;
font-weight: 800;
/* Text style for &quot;Welcome  B&quot; */
letter-spacing: 0.6px;</string>
      </property>
      <property name="text">
       <string>device_id</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>310</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>470</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_4 {
	background-image:/img/Rectangle_4.png;
width: 101px;
height: 48px;
border-radius: 4px;
background-color: #5e77ff;


color: #ffffff;
font-family: &quot;SF UI Display&quot;;
font-size: 14px;
font-weight: 600;
line-height: 20px;
/* Text style for &quot;Login&quot; */
letter-spacing: 0.5px;
}

#pushButton_4:hover {
	background:-webkit-gradient(linear, left top, left bottom, color-stop(0.05, #019ad2), color-stop(1, #33bdef));
	background:-moz-linear-gradient(top, #019ad2 5%, #33bdef 100%);
	background:-webkit-linear-gradient(top, #019ad2 5%, #33bdef 100%);
	background:-o-linear-gradient(top, #019ad2 5%, #33bdef 100%);
	background:-ms-linear-gradient(top, #019ad2 5%, #33bdef 100%);
	background:linear-gradient(to bottom, #019ad2 5%, #33bdef 100%);

	background-color:#019ad2;
}
#pushButton_4:active {
	position:relative;
	top:1px;
}</string>
   </property>
   <property name="text">
    <string>New User? Sign Up</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>label_9</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>status</zorder>
  <zorder>label_4</zorder>
  <zorder>round</zorder>
  <zorder>label_6</zorder>
  <zorder>label_8</zorder>
  <zorder>lineEdit</zorder>
  <zorder>label_10</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>label_11</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
